app/modules/ads/setup.js

```diff
@@ -127,43 +127,45 @@ export const adsSetup = {
       context.set('src', 'test');
     }
 
-    instantConfig.isGeoEnabled('wgAdDriverLABradorTestCountries');
+    instantConfig.get('icLABradorTest');
     context.set('slots', slots.getContext());
 
     if (!adsContext.targeting.hasFeaturedVideo && adsContext.targeting.pageType !== 'search') {
       context.push('slots.top_leaderboard.defaultSizes', [2, 2]);
     }
 
-    const stickySlotsLines = instantConfig.get('wgAdDriverStickySlotsLines');
+    const stickySlotsLines = instantConfig.get('icStickySlotLineItemIds');
 
     if (stickySlotsLines && stickySlotsLines.length) {
       context.set('templates.stickyTLB.lineItemIds', stickySlotsLines);
       context.push('slots.top_leaderboard.defaultTemplates', 'stickyTLB');
     }
 
-    context.set('state.disableTopLeaderboard', instantConfig.isGeoEnabled('wgAdDriverCollapseTopLeaderboardMobileWikiCountries'));
+    context.set('state.disableTopLeaderboard', instantConfig.get('icCollapseTopLeaderboard'));
     context.set('state.deviceType', utils.client.getDeviceType());
 
     context.set('options.billTheLizard.cheshireCat', adsContext.opts.enableCheshireCat);
     context.set('options.billTheLizard.config', instantConfig.get('wgAdDriverBillTheLizardConfig'));
 
-    context.set('options.video.moatTracking.enabled', instantConfig.isGeoEnabled('wgAdDriverPorvataMoatTrackingCountries'));
-    context.set('options.video.moatTracking.sampling', instantConfig.get('wgAdDriverPorvataMoatTrackingSampling'));
+    context.set('options.video.moatTracking.enabled', instantConfig.get('icPorvataMoatTracking'));
+    //  During moving variables from WikiFactory to ICBM, options.video.moatTracking.sampling
+    //  was set to 100 since in ad-engine it's being used in getMoatTrackingStatus() function.
+    context.set('options.video.moatTracking.sampling', 100);
     context.set('options.video.iasTracking.enabled', instantConfig.get('icIASVideoTracking'));
 
-    context.set('options.video.playAdsOnNextVideo', instantConfig.isGeoEnabled('wgAdDriverPlayAdsOnNextFVCountries'));
-    context.set('options.video.adsOnNextVideoFrequency', instantConfig.get('wgAdDriverPlayAdsOnNextFVFrequency'));
-    context.set('options.video.isMidrollEnabled', instantConfig.isGeoEnabled('wgAdDriverFVMidrollCountries'));
-    context.set('options.video.isPostrollEnabled', instantConfig.isGeoEnabled('wgAdDriverFVPostrollCountries'));
+    context.set('options.video.playAdsOnNextVideo', !!instantConfig.get('icFeaturedVideoAdsFrequency'));
+    context.set('options.video.adsOnNextVideoFrequency', instantConfig.get('icFeaturedVideoAdsFrequency'));
```

Comment:
@Loczi94 could you revisit the value of `icFeaturedVideoAdsFrequency` in ICBM? Shouldn't we set it to `3`?

Reply:
`wgAdDriverPlayAdsOnNextFVFrequency` changed value from 1 to 3 after I added `icFeaturedVideoAdsFrequency` to ICBM. I'll update the value.